### PR TITLE
feat: [FC-0047] move the certificate view from the banner to the message section

### PR DESCRIPTION
**Pull request by volodymyr-chekyrta**
This PR is related to the FC-0047 course home redesign and includes the new certificate message block.

Design:
https://www.figma.com/design/iZ56YMjbRMShCCDxqrqRrR/Open-edX-Mobile-App-All-Screens-v2.1?node-id=14957-309221&t=e1C9Uxns0YlV3sHh-0

Demo:
| Phone | Tablet |
| ------------- | ------------- |
| image | image |

Changes:
- Introduced new composable function CourseMessage.
- Removed unused CourseImageHeader and string resources.

Checklist:
✅ Certificate message appears if the course certificate is earned
✅ A11y. The message block has a big clickable area and works as 1 object for the TalkBack

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @volodymyr-chekyrta! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by volodymyr-chekyrta** [comment]
@HamzaIsrar12 @k1rill, Kindly review the changes made in this pull request. Thank you.

**Comment by omerhabib26** [comment]
Reviewing

File: course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt [hunk 1]
```diff
@@ -110,89 +98,6 @@ import subtitleFile.TimedTextObject
...
-                Modifier
-                    .fillMaxSize()
-                    .clip(MaterialTheme.appShapes.cardShape)
-                    .background(MaterialTheme.appColors.certificateForeground),
-                horizontalAlignment = Alignment.CenterHorizontally,
-                verticalArrangement = Arrangement.Center
-            ) {
-                Icon(
-                    modifier = Modifier.testTag("ic_congratulations"),
-                    painter = painterResource(id = R.drawable.ic_course_completed_mark),
```
Comment (HamzaIsrar12) [inline course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt:162]: We can remove the drawable as well.

**Review by HamzaIsrar12 — APPROVED** [review]
One minor nit only. 👍🏻

File: course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt (hunk 1 above)
Comment (volodymyr-chekyrta) [reply course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt:162]: That makes sense; thank you! ✅

Comment (volodymyr-chekyrta) [reply course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt:162]: @HamzaIsrar12, could you please re-approve after the change?

**Review by HamzaIsrar12 — APPROVED** [review]
LGTM 🏎️

File: course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt (hunk 1 above)
Comment (HamzaIsrar12) [reply course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt:162]: Done ✨

**Comment by openedx-webhooks** [comment]
@volodymyr-chekyrta 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-app-android&entry.867055334=https://github.com/openedx/openedx-app-android/pull/321&entry.1484655318=https://github.com/volodymyr-chekyrta&entry.752974735=2024-05-14+19:58&entry.1917517419=2024-05-20+07:51&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**